<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="menu_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>980</y>
      <width>1901</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QPushButton" name="home_btn">
     <property name="geometry">
      <rect>
       <x>1690</x>
       <y>1</y>
       <width>160</width>
       <height>62</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../alfa_CR6_ui/images/alfa_logo.png</normaloff>../alfa_CR6_ui/images/alfa_logo.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>60</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">menu_btn_group</string>
     </attribute>
    </widget>
    <widget class="QPushButton" name="order_btn">
     <property name="geometry">
      <rect>
       <x>1500</x>
       <y>1</y>
       <width>160</width>
       <height>62</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../alfa_CR6_ui/images/barcode_C128.png</normaloff>../alfa_CR6_ui/images/barcode_C128.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>60</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">menu_btn_group</string>
     </attribute>
    </widget>
    <widget class="QPushButton" name="keyboard_btn">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>1</y>
       <width>160</width>
       <height>62</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../alfa_CR6_ui/images/keybd.png</normaloff>../alfa_CR6_ui/images/keybd.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>60</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">menu_btn_group</string>
     </attribute>
    </widget>
    <widget class="QPushButton" name="browser_btn">
     <property name="geometry">
      <rect>
       <x>1280</x>
       <y>1</y>
       <width>160</width>
       <height>62</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../alfa_CR6_ui/images/autofinish.png</normaloff>../alfa_CR6_ui/images/autofinish.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>60</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">menu_btn_group</string>
     </attribute>
    </widget>
    <widget class="QPushButton" name="global_status_btn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>1</y>
       <width>160</width>
       <height>62</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../alfa_CR6_ui/images/green.svg</normaloff>../alfa_CR6_ui/images/green.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>60</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stacked_widget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>960</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(&quot;./../alfa_CR6_ui/images/machine_r.png&quot;);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <widget class="QWidget" name="home_page">
     <property name="styleSheet">
      <string notr="true">background-image:&quot;./../alfa_CR6_ui/images/machine_r.png&quot;;</string>
     </property>
     <widget class="QPushButton" name="service_1_btn">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>190</y>
        <width>320</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:#FFFFFF</string>
      </property>
      <property name="text">
       <string>service 1</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">service_btn_group</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="service_3_btn">
      <property name="geometry">
       <rect>
        <x>922</x>
        <y>190</y>
        <width>320</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:#FFFFFF</string>
      </property>
      <property name="text">
       <string>service 3</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">service_btn_group</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="service_6_btn">
      <property name="geometry">
       <rect>
        <x>1308</x>
        <y>560</y>
        <width>320</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:#FFFFFF</string>
      </property>
      <property name="text">
       <string>service 6</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">service_btn_group</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="service_4_btn">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>560</y>
        <width>320</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:#FFFFFF</string>
      </property>
      <property name="text">
       <string>service 4</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">service_btn_group</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="service_2_btn">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>560</y>
        <width>320</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:#FFFFFF</string>
      </property>
      <property name="text">
       <string>service 2</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">service_btn_group</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="service_5_btn">
      <property name="geometry">
       <rect>
        <x>1308</x>
        <y>190</y>
        <width>320</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:#FFFFFF</string>
      </property>
      <property name="text">
       <string>service 5</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">service_btn_group</string>
      </attribute>
     </widget>
    </widget>
    <widget class="QWidget" name="order_page">
     <property name="styleSheet">
      <string notr="true">background:#FFFFFF</string>
     </property>
     <widget class="QTableView" name="order_table">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>1591</width>
        <height>931</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="new_btn">
      <property name="geometry">
       <rect>
        <x>1620</x>
        <y>830</y>
        <width>231</width>
        <height>64</height>
       </rect>
      </property>
      <property name="text">
       <string>new</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="browser_page">
     <property name="styleSheet">
      <string notr="true">background:#FFFFFF</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="service_btn_group"/>
  <buttongroup name="menu_btn_group">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
